defopt.py

```diff
@@ -54,16 +54,18 @@ def run(*funcs, **kwargs):
     :param Callable funcs: Function or functions to process and run
     :param parsers: Dictionary mapping types to parsers to use for parsing
         function arguments.
-    :type parsers: Dict[type, Callable[[str], type]]
+    :type parsers: Optional[Dict[type, Callable[[str], type]]]
```

Comment:
Is it necessary to mark this as optional? The [documentation](https://docs.python.org/3/library/typing.html#typing.Optional) for it says "An optional argument with a default needn’t use the Optional qualifier on its type annotation". Is there anything about this argument in particular that distinguishes it from `short` or `argv`?